.github/workflows/dispatch.yml

```
@@ -0,0 +1,38 @@
+name: Deploy on Events
+
+on:
+  repository_dispatch:
+    types: [dev-deploy]
+  workflow_dispatch:
+    inputs:
+      sc-branch:
```

would be a good time to name to something else without "sc" 
```suggestion
      branch_name:
```

---

Add this to an batch with the place where it was used